packages/framework/content.ts

```diff
@@ -5,7 +5,7 @@ import {addHistoryChangeListener} from "./history";
 const pageLoadingKey = Symbol("pageLoading");
 const loadingPromiseKey = Symbol("pageLoadingPromise");
 
-export class XContent extends Component {
+export class XContent extends Component<undefined, undefined> {
```

Comment:
Component<undefined, undefined> <- probably it must be by default?

Reply:
fixed